app/views/teams/users/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1 class="usa-display">Add New user</h1>
+
+<%= simple_form_for(@user,
+    html: { autocomplete: 'off', role: 'form' },
+    url: team_users_path(@team)) do |form| %>
+  <%= render 'form', form: form %>
+  <br/>
+  <%= form.button :submit, 'Add', :class => "usa-button" %>
```

Comment:
If you wrap this button and the one below in a div you can probably [add some margin with a utility class ](https://designsystem.digital.gov/utilities/margin-and-padding/)